homeassistant/components/sonos/media_player.py

```diff
@@ -236,6 +237,7 @@ async def async_service_handle(service_call: ServiceCall) -> None:
             vol.Optional(ATTR_NIGHT_SOUND): cv.boolean,
             vol.Optional(ATTR_SPEECH_ENHANCE): cv.boolean,
             vol.Optional(ATTR_STATUS_LIGHT): cv.boolean,
+            vol.Optional(ATTR_EQ_BASS): cv.latitude,
```

Comment:
this config validation is wrong. Need also match with the min, max

Reply:
The underlying library clamps the provided value with a setter.

Reply:
What is the best choice here. cv.small_float is my assumption but the sonos setting has to be a whole number from -10 to +10 and of course small_float allows decimals. 

My first contribution please feel free to point me in the direction of docs for a way around this

Reply:
https://github.com/alecthomas/voluptuous

Reply:
Swapped ot int